sig-platform/community-donated-plugins.md

```diff
@@ -0,0 +1,13 @@
+# Community Donated Plugins
+
+The [`spinnaker-plugins`](https://github.com/spinnaker-plugins) GitHub organization is a holding repository for unofficial, community-contributed Spinnaker plugins.
+Typical cases for moving a plugin to this organization would include companies who require CLAs for contributions to their main GitHub organizations, but do not want to include that overhead for contributorship to the plugin itself. **Donating a plugin to the `spinnaker-plugins` organization is never required.**
+
+Plugin authors looking to transfer ownership of their plugins to the `spinnaker-plugins` organization will:
+
+- Continue to be the maintainers of their plugin.
+- Will be granted `write` access to the repository.
+
+To start the process of transferring a plugin to `spinnaker-plugins`, [create an issue](https://github.com/spinnaker/governance/issues/new/choose) in the governance repository.
+
+Once approved, a member of the Technical Oversight Committee will be in contact for arranging transfer and setup of the repository.
```

Comment:
What, if any, license conditions apply to plugins donated here? Is it acceptable for them to have any kind of license whatsoever?

Reply:
I think it's fair to require that donated plugins declare a license. 

Given that these plugins are not distributed alongside a Spinnaker release, I'm not sure we need to limit what the actual license is though.

Reply:
I was debating dictating licenses, but opted against. I do like adding that they must be licensed in some form, but won't make it a requirement to what it is.

Reply:
In that case, perhaps add something along the lines of "Donated code should/must explicitly declare a license."?

Reply:
I see the update now - thanks!